.travis.yml

```diff
@@ -0,0 +1,70 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+
+env:
+  global:
+    # h5py: pip install work-around for "mpi.h: No such file or directory"
+    - C_INCLUDE_PATH=/usr/lib/openmpi/include
+  matrix:
+    - USE_ADIOS=OFF
+  # - USE_ADIOS=SERIAL
+  # - USE_ADIOS=MPI
+  # - USE_ADIOS=OFF USE_CUDA=ON
+
+addons:
+  apt:
+    packages:
+      - openmpi-bin
+      - libopenmpi-dev
+      - libhdf5-openmpi-dev
+    # - libadios-dev
```

Comment:
disabled, waiting for https://github.com/travis-ci/apt-package-whitelist/issues/2187